index.js

```diff
@@ -503,12 +518,12 @@ class SortableListView extends React.Component {
   }
 
   scrollTo = (...args) => {
-    if (!this.refs.list) return;
+    if (!this.refs.list) return
```

Comment:
Really stupid nit: Would this cause complaints from eslint?

Reply:
Agreed - but it was done upstream - so thinking we should just keep it out for now https://github.com/hudl/react-native-sortable-listview/pull/1/commits/ad6afb0ba712c26c08d18b94390154aa3958a3a0

Reply:
Ignore this: was added in a different commit that was merged into this PR https://github.com/hudl/react-native-sortable-listview/pull/1/commits/ad6afb0ba712c26c08d18b94390154aa3958a3a0